lib/app.rb

```diff
@@ -16,11 +16,11 @@ def build_board
       @simulator.setup_board(board_rows_input, baord_columns_input)
 
       @display.build_board_complete_message
-      @simulator.board.print_board
+      @display.print_board(@simulator.board.board_with_axis)
 
       return if @simulator.board
     rescue ArgumentError => e
-      puts e
+      @display.print_error(e)
```

Comment:
👍 - this is good, probably very generic and could be reused in other games that use our Display class